doc/dev/python_scheduling/collect_pointwise_data.py

```diff
@@ -0,0 +1,378 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# Owner(s): ["module: nvfuser"]
+
+import torch
+import itertools
+import numpy as np
+from nvfuser import FusionCache, FusionDefinition, SchedulerType
+
+# ============================ Description ============================
```

Comment:
Can we also add a comment on how to run this script, and the output it will generate?
Have you used this script to generate data for autotuning?